core/src/main/resources/error/error-classes.json

```diff
@@ -1418,7 +1418,7 @@
       "It is not allowed to add database prefix `<database>` for the TEMPORARY view name."
     ]
   },
-  "_LEGACY_ERROR_TEMP_0055" : {
+  "UNCLOSED_BRACKETED_COMMENT" : {
```

Comment:
Don't think you can rename in place. Error class names should be sorted in the file.

Reply:
Oops, my mistake. Thanks for the correction!